platform/projects_and_organizations/project_settings.mdx

```diff
@@ -8,15 +8,18 @@ import { ScreenshotWrapper } from '../shared/_ScreenshotWrapper';
 
 In project settings, you can edit the basic properties of your project.
 
-- **project name**
-- **description** - displayed in the project dashboard (can be formatted with Markdown)
-- **base language** - the language that is used as a base for translating to other languages
-- **default namespace** - preselected namespace when creating a new key
+- **Project Name**
+- **Description** - displayed in the project dashboard (can be formatted with Markdown)
+- **Base language** - the language that is used as a base for translating to other languages
+- **Default namespace** - preselected namespace when creating a new key
+
+To use namespaces in your project, you need to have namespaces enabled. Toggle "Use namespaces" to choose your default namespace. 

```

Comment:
There is actually `Use namespaces` checkbox above the `default namespace`, so it should be added above.

Also I would add separate section called `Enabling namespaces`